src/app/main.cpp

```diff
@@ -1434,7 +1434,7 @@ int main( int argc, char *argv[] )
         break;
       }
 
-      coords[i] = myInitialExtent.midRef( posOld, pos - posOld ).toDouble( &ok );
+      coords[i] = QStringView{myInitialExtent}.mid( posOld, pos - posOld ).toDouble( &ok );
```

Comment:
do we need to explicitly use string view? I thought it was transparent...

Reply:
Copied from here: https://doc-snapshots.qt.io/qt6-dev/qtcore-changes-qt6.html#the-qstringref-class

Reply:
Ok, I have some learning to do :D

Reply:
```suggestion
      coords[i] = QStringView{myInitialExtent}.mid( posOld, pos - posOld ).toString().toDouble( &ok );
```